### DISCUSSION TITLE
makefile

### DISCUSSION THREAD

**Discussion by soreettihussein** [omnetpp/omnetpp#1179] [Q&A]
Hello! I am trying to build nesting-master with inet4.1.2  I received the following error message, please anyone can give me suggestions on it
make MODE=release -j3 all 
make[1]: Entering directory '/home/user/omnetpp-5.5.1/samples/nesting-master/src' Creating shared library: ./out/clang-release/src/libnesting.so
/usr/bin/ld: cannot find -lINET: No such file or directory
 thank you!

**Comment by rhornig** [comment]
Either INET is not built successfully or it's not correctly configured in the nesting project (i.e. the nesting build process does not find it).

Why don't you try opp_env? 

    opp_env install nesting-latest

**Reply by soreettihussein** [reply]
INET framework built successfully. please give explanation how to configure nesting project and try opp_env install nesting-latest
